### PR TITLE
Create Asia-Pacific-community-call.md

### PR DESCRIPTION
**Pull request by germonprez**
Signed-off-by: Matt Germonprez <germonprez@gmail.com>

Creating the Asia Pacific participate information

### CODE REVIEW

File: Participate/Asia-Pacific-community-call.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+Asia Pacific Community Call
+
+This working group aims to connect with community members in the Asia Pacific region regarding open source community health. Discussions will focus on recaping CHAOSS work as well as identifying new areas of interest. The working group is, of course, open to everyone from any global region. 
+
+The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 15:00 CET, check your local time) via Zoom -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
```
Comment (GeorgLink) [inline Participate/Asia-Pacific-community-call.md:6]: Please turn "check your local time" into a link to https://arewemeetingyet.com with the meeting times setup

File: Participate/Asia-Pacific-community-call.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+Asia Pacific Community Call
```
Comment (GeorgLink) [inline Participate/Asia-Pacific-community-call.md:2]: Please format Line 2 as a heading. 

```suggestion
### Asia Pacific Community Call
```

**Comment by germonprez** [comment]
Is this what you're looking for?

**Comment by GeorgLink** [comment]
When you go to the arewemeetingyet website, you can set up a recurring meeting. Once you are on that page, you can copy the URL and use that in our website. That's what we have done for all the other calls.

**Comment by germonprez** [comment]
I did it but it looks completely wrong :)

**Comment by klumb** [comment]
This will partially resolve #324

File: Participate/Asia-Pacific-community-call.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+
+### Asia Pacific Community Call
+
+This working group aims to connect with community members in the Asia Pacific region regarding open source community health. Discussions will focus on recaping CHAOSS work as well as identifying new areas of interest. The working group is, of course, open to everyone from any global region. 
+
+The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 15:00 CET, [check your local time](https://arewemeetingyet.com/(GMT-5:00)%20Central%20Time%20(US%20and%20Canada)/2020-06-17/08:00/b/CHAOSS%20Asia%20Pacific%20Community%20Call)) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
```
Comment (GeorgLink) [inline Participate/Asia-Pacific-community-call.md:6]: This should fix the arewemeetingyet.com link
and add the Zoom link
and refer to normal China time

```suggestion
The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 21:00 (UTC+8), [check your local time](https://arewemeetingyet.com/Chicago/2020-06-17/08:00/b/CHAOSS%20Asia%20Pacific%20Community%20Call#eyJ1cmwiOiJodHRwczovL3Vub21haGEuem9vbS51cy9qLzcyMDQzMTI4OCJ9)) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
```

**Comment by germonprez** [comment]
thanks @GeorgLink

File: Participate/Asia-Pacific-community-call.md [hunk 4]
```diff
@@ -3,4 +3,4 @@
 
 This working group aims to connect with community members in the Asia Pacific region regarding open source community health. Discussions will focus on recaping CHAOSS work as well as identifying new areas of interest. The working group is, of course, open to everyone from any global region. 
 
-The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 21:00 (UTC+8), [check your local time](https://arewemeetingyet.com/Chicago/2020-06-17/08:00/b/CHAOSS%20Asia%20Pacific%20Community%20Call#eyJ1cmwiOiJodHRwczovL3Vub21haGEuem9vbS51cy9qLzcyMDQzMTI4OCJ9)) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
+The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 21:00 (UTC+8), [check your local time](https://arewemeetingyet.com/(GMT-5:00)%20Central%20Time%20(US%20and%20Canada)/2020-06-17/08:00/b/CHAOSS%20Asia%20Pacific%20Community%20Call)) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
```
Comment (GeorgLink) [inline Participate/Asia-Pacific-community-call.md:6]: It looks like there was a regression in the last commit.

This should fix it again:

```suggestion
The Asia Pacific community call meets every other Wednesday at 8:00am CT (usually 21:00 (UTC+8), [check your local time](https://arewemeetingyet.com/Chicago/2020-06-17/08:00/b/CHAOSS%20Asia%20Pacific%20Community%20Call)) via [Zoom](https://unomaha.zoom.us/j/720431288) -- [Agenda and Meeting Minutes](https://docs.google.com/document/d/17S89h-0MpMA0fzkxv16LRgK8Nh1Qov7lammjNnPk16E/edit)
```

**Comment by GeorgLink** [comment]
@klumb can you add a module for this on the Participate website?

**Comment by klumb** [comment]
There are two other new modules that are going on to the website. I am going to add them all at the same time. Details are in #324

**Pull request merged**